sandbox.sh

```diff
@@ -21,32 +18,39 @@ export PATH
 
 [ "$USE_NIX" ] && dune build @install
 
+# Use tezos-client to interact with the node.
+# See https://tezos.gitlab.io/introduction/howtouse.html#client
 tezos-client() {
   docker exec -t deku_flextesa tezos-client "$@"
 }
 
+# Necessary to write tezos smart contracts
 ligo() {
   docker run --rm -v "$PWD":"$PWD" -w "$PWD" ligolang/ligo:0.28.0 "$@"
 }
 
-if [ $mode  = "docker" ]
-then
+if [ $mode = "docker" ]; then
   RPC_NODE=http://flextesa:20000
 else
   RPC_NODE=http://localhost:20000
 fi
 
+# Tezos secret key?
 # This secret key never changes.
 SECRET_KEY="edsk3QoqBuvdamxouPhin7swCvkQNgq4jP5KZPbwWNnwdZpSpJiEbq"
 
+# Folder containing the data of each nodes (identity.json)
 DATA_DIRECTORY="data"
 
+# Names of the validators, see content of data folder
 VALIDATORS=(0 1 2)
 
+# Helper to print message
 message() {
   echo "=========== $* ==========="
 }
 
+# Create validators.json containing the URL and the tezos address of each validator (see data folder)
```

Comment:
This comment is incorrect: the validator addresses are not Tezos addresses (they begin with `tz1` or `tz2` but this is to support Tezos' hashing scheme - they don't correlate to actual Tezos wallets).

Reply:
Oh ok, thanks.